backend/strapi-sync/create-single-item-campgrounds.js

```diff
@@ -0,0 +1,1846 @@
+    items: [{ featureStrapiId: 479, featureName: "All sites" }],
+  },
+  {
+    parkId: "993",
+    campgroundName: "Wasa Lake Campground",
+    items: [{ featureStrapiId: 480, featureName: "All sites" }],
+  },
+  {
+    parkId: "329",
+    campgroundName: "Wedgemount Lake Campground",
+    items: [{ featureStrapiId: 482, featureName: "All sites" }],
+  },
+  {
+    parkId: "221",
+    campgroundName: "West Curme Island",
+    items: [{ featureStrapiId: 483, featureName: "All sites" }],
+  },
+  {
+    parkId: "1007",
+    campgroundName: "Whiskers Point Campground",
+    items: [{ featureStrapiId: 486, featureName: "All sites" }],
+  },
+  {
+    parkId: "1009",
+    campgroundName: "White Lake Campground",
+    items: [{ featureStrapiId: 487, featureName: "All sites" }],
+  },
+  {
+    parkId: "1013",
+    campgroundName: "White River Campground",
+    items: [{ featureStrapiId: 488, featureName: "All sites" }],
+  },
+  {
+    parkId: "832",
+    campgroundName: "Whitworth Horse Camp",
+    items: [{ featureStrapiId: 491, featureName: "All sites" }],
+  },
+  {
+    parkId: "717",
+    campgroundName: "Widgeon Creek Campground",
+    items: [{ featureStrapiId: 751, featureName: "All sites" }],
+  },
+  {
+    parkId: "329",
+    campgroundName: "Wilderness Camping Area",
+    items: [{ featureStrapiId: 492, featureName: "All sites" }],
+  },
+  {
+    parkId: "487",
+    campgroundName: "Woodbury Cabin",
+    items: [{ featureStrapiId: 494, featureName: "All sites" }],
+  },
+  {
+    parkId: "347",
+    campgroundName: "Woodlands groupsite",
+    items: [{ featureStrapiId: 613, featureName: "All sites" }],
+  },
+  {
+    parkId: "1029",
+    campgroundName: "Yahk Campground",
+    items: [{ featureStrapiId: 495, featureName: "All sites" }],
+  },
+  {
+    parkId: "1036",
+    campgroundName: "sx\u030c\u02b7\u0259x\u030c\u02b7nitk\u02b7 Campground",
+    items: [{ featureStrapiId: 446, featureName: "All sites" }],
+  },
+  {
+    parkId: "1035",
+    campgroundName: "s\u1e83i\u1e83s Campground",
+    items: [{ featureStrapiId: 380, featureName: "All sites" }],
+  },
+  {
+    parkId: 1035,
+    campgroundName: "s\u1e83i\u1e83s overflow camping",
+    items: [{ featureStrapiId: 319, featureName: "All sites" }],
+  },
+];
+
+async function createCampground(item) {
+  const park = await getItemByAttributes(Park, {
+    strapiId: parseInt(item.parkId, 10),
+  });
+
+  // create campground with FK to Park
+  const data = {
+    name: item.campgroundName,
+    parkId: park.id,
+  };
+
+  const campground = await createModel(Campground, data);
+
+  // get feature by strapiID
+  const feature = await getItemByAttributes(Feature, {
+    strapiId: item.items[0].featureStrapiId,
+  });
+
+  feature.campgroundId = campground.id;
+  feature.name = "All sites";
+  await feature.save();
+}
+
+async function createSingleItemsCampgrounds(items) {
+  await Promise.all(items.map(async (item) => await createCampground(item)));
```

Comment:
Technically we don't need the async/await in the callback, because we just want it to return a Promise anyway. It won't make a difference besides making the line a little shorter if you want to change it